### ISSUE TITLE
Adversarial loss!

### ISSUE THREAD

**Issue by l-cr** [Leminhbinh0209/FinetuneVAE-SD#4]
Hi! Why not use adversarial loss？

**Comment by Leminhbinh0209** [comment]
I follow this post from HuggingFace: https://huggingface.co/stabilityai/sd-vae-ft-mse-original/blob/main/README.md

The quality of original model is already good,  just because I tried with smaller image it generates some visible artifacts, thus I just used simple losses. If fine-tune on a total new image domain, using adversarial loss is needed, indeed.
